roles/ansible/tower/config-ansible-tower/tasks/install-cert.yml

```diff
@@ -0,0 +1,18 @@
+---
```

Comment:
great - this is good, but the steps will now be done twice as they also still exists in the `install.yml`